core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -1574,7 +1574,6 @@ class DAGScheduler(
   }
 
   def stop() {
-    logInfo("Stopping DAGScheduler")
```

Comment:
this is done as part of https://github.com/apache/spark/pull/9603#issuecomment-159190700 but it is way too small to deserve its own pr.